packages/ui-kit/src/components/rss/rss-feeds.tsx

```diff
@@ -101,6 +102,7 @@ const transformData = (data: FlatRssEntry[][]) => {
   const tableData: FlatRssEntry[] = data
     .flat()
     .reduce<FlatRssEntry[]>((list, entry) => {
+      if (!lastEntryOfList.pubDate) return [...list, entry];
```

Comment:
In case we don't have feeds with enough entries (like in the smoke-test atm). Not really important for the docs use-case.